providers/class-two-factor-fido-u2f-admin.php

```diff
@@ -324,7 +335,7 @@ public static function wp_ajax_inline_save() {
 		}
 
 		foreach ( $security_keys as &$key ) {
-			if ( $key->keyHandle === $_POST['keyHandle'] ) {
+			if ( $key->key_handle === $_POST['key_handle'] ) {
```

Comment:
This would need to be reverted if we're keeping the vendor libraries away from WPCS.